app/data/views.py

```diff
@@ -198,6 +202,22 @@ def toddow(self, request):
         return Response(counted)
 
     @list_route(methods=['get'])
+    def recent_counts(self, request):
+        """ Return the recent record counts for 30, 90, 365 days """
+        now = datetime.datetime.now(tz=pytz.timezone(settings.TIME_ZONE))
+        qs = self.get_filtered_queryset(request).filter(occurred_from__lte=now)
+        durations = {
+            'month': 30,
+            'quarter': 90,
+            'year': 365,
+        }
+
+        counts = {label: qs.filter(occurred_from__gte=(now - datetime.timedelta(days=days))).count()
+                  for label, days in durations.items()}
```

Comment:
Is there a nice way to do this with one query instead of three?  It seems similar to the other aggregations happening in the other Record views, but I don't have much experience with those tools and my attempts met with failure.